src/Controller/UI/LandingController.php

```diff
@@ -55,4 +56,15 @@ public function researchGroupLand(ResearchGroup $researchGroup, InformationProdu
             ]
         );
     }
+
+    #[Route('/funding-organization/about/{fundingOrganization}', name: 'app_funding_organization_land')]
```

Comment:
The ticket did mention that this needs to be restricted to just DRPM/(and FOPM).

Reply:
This is just a preview, and this PR should be in draft.